CommentingPractice/comment-practice.js

```diff
@@ -22,7 +22,7 @@ import * as adrianna from './services/sum.js';
 import * as glory from './model/rome.js';
 import TeamMate from './model/teammate.js'; 
```

Comment:
In order to utilize TeamMate, we need access to its super-class "Team". We can fix this by writing something like 

```
import * as T from './model/teammate.js';
```

and invoking constructors like this

```
const member = new T.TeamMate("Sanjana", "Yadav");
```